src/python/crash_analysis/crash_analyzer.py

```diff
@@ -348,8 +349,11 @@ def is_security_issue(crash_stacktrace, crash_type, crash_address):
   if not crash_type:
     return False
 
+  is_lkl = lkl_constants.LINUX_KERNEL_MODULE_STACK_TRACE in crash_stacktrace
   # Check signal types, some of them indicate not a security bug.
-  if has_signal_for_non_security_bug_type(crash_stacktrace):
+  # LKL stacks may have ABRTs and gsignal etc in them on the libfuzzer side,
+  # This should not affect whether they are marked security.
+  if not is_lkl and has_signal_for_non_security_bug_type(crash_stacktrace):
```

Comment:
this makes the logic a bit difficult to follow, and couples this code to lkl which is not ideal. We should just be relying on the crash_type and other testcase information here. 

What's the crash type for these bugs? Is it 'Kernel failure' ? Can you just add a crash_type check above to return True?